src/esp/gfx/Drawable.h

```diff
@@ -94,14 +94,17 @@ class Drawable : public Magnum::SceneGraph::Drawable3D {
    */
   virtual void setLightSetup(
       CORRADE_UNUSED const Magnum::ResourceKey& lightSetup){};
-
   /**
    * @brief the the scene node
    */
   virtual scene::SceneNode& getSceneNode() const { return node_; }
 
   /** @brief get the GL mesh */
-  Magnum::GL::Mesh& getMesh() const { return mesh_; }
+  Magnum::GL::Mesh& getMesh() const {
+    CORRADE_ASSERT(mesh_ != nullptr,
+                   "Attempting to get the GL mesh when none exists", *mesh_);
```

Comment:
You are missing the class name and function name in the error message.
"Drawable::getMesh(): ..."